docs/Introduction/architecture-overview/architecture-decentralized-model.md

```diff
@@ -63,14 +63,14 @@ Learn how to create a consumer contract to [Get the Latest Price](../get-the-lat
 
 ## Proxy
 
-Proxy contracts are on-chain proxies that store the most up-to-date Aggregator for a particular price feed. Using proxies enables the underlying Aggregator to be upgraded without any interruption of service for consuming contracts.
+Proxy contracts are on-chain proxies that store the most up-to-date Aggregator for a particular data feed. Using proxies enables the underlying Aggregator to be upgraded without any interruption of service for consuming contracts.
 
 See the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
 
 ## Aggregators
 
 Aggregators are the contracts that receive periodic price updates from multiple Oracles. They aggregate and store the current price on-chain so that consumers can obtain the latest price and act upon it within the same transaction.
```

Comment:
```suggestion
Aggregators are the contracts that receive periodic data updates from multiple Oracles. They aggregate and store data on-chain so that consumers can retrieve it and and act upon it within the same transaction.
```